.github/workflows/wrapper-upgrade-execution.yml

```diff
@@ -0,0 +1,41 @@
+name: Execute Wrapper Upgrade
+
+on:
+  schedule:
+    - cron:  '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+  upgrade_wrapper:
+    name: Execution
+    runs-on: ubuntu-latest
+    env:
+      GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@cb4264d3319acaa2bea23d51ef67f80b4f775013
+        with:
+          gpg_private_key: ${{ secrets.GH_BOT_PGP_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GH_BOT_PGP_PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          git_config_global: true
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Clean with Gradle
```

Comment:
IMO, we can remove the clean step.

Reply:
Yes, but still needs to execute a clean since so our wrapper upgrade plugin will be able to clone (if not it fails because the directory already exists in `build/git-clones`

Reply:
I don't understand.

Reply:
I've removed the `clean`, it should not be a problem if GH action state restore does not interfere